### ISSUE TITLE
Fisherman throws out line but instantly  retrieves after 2 seconds punches air and then repeats.

### ISSUE THREAD

**Issue by MertFz** [ldtteam/minecolonies#6061]
### Prerequisites 

- [x ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [ x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x ] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 
- Minecraft Version: 1.15.2
- MineColonies Version: minecolonies-0.13.438-ALPHA
- Structurize Version: structurize-0.13.94-ALPHA
- Related Mods and their Versions: Aquaculture-1.15.2-2.0.23.jar, Upgrade-Aquatic-1.15.2-1.7.1.jar ( its the modpack craft to exile [Dissonance])

### Expected behavior
The Fisherman throws his line in the water waits until he catches fish pulls back and continues.

Actual behavior
The Fisherman throws his line in the water he instantly pulls it back after 2 sec without catching anything and the rod loses durability. Then he punches the air once and continues to repeat the cycle.

![image](https://user-images.githubusercontent.com/28735764/98726091-5438b080-2396-11eb-8301-2a9e88211275.png)
[Video](https://kapwi.ng/w/5ya0Rh3h)

### Steps to reproduce the issue
1. Download latest version of Craft to Exile [Dissonance] modpack
2. Start a world
3. buid the Fishermans hut
4. Assign Fisherman
5. give him the fishing rod and watch

**Comment by Raycoms** [comment]
Can you try without those two: Aquaculture-1.15.2-2.0.23.jar, Upgrade-Aquatic-1.15.2-1.7.1.jar?

**Comment by MertFz** [comment]
I threw out both the mods and the Fisherman is stil doing the same thing

**Comment by Raycoms** [comment]
Double checked this, there must be another mod in the pack then that is causing this.

**Comment by mahjerion** [comment]
Hey, CtE packdev here - saw this while I was looking for another issue I was having. Thought I'd chime in. The only mod I updated that might have affected this when it started happening was Aquaculture from 2.0.16 to 2.0.23. But even after going back to 2.0.16 or removing it, the issue persisted. However, another thing I did was update to Forge 31.2.45 from 31.2.36. I'm _pretty_ sure it has to do with this as the other changes I've made probably don't affect the fishermen. What version of Forge is the current alpha being built on?
